src/reducers/BackendModeSignedInStatus.js

```diff
@@ -1,6 +1,5 @@
-import axios from 'axios';
```

Comment:
Thanks for removing this. I added this in because the login wasn't working and was trying a different library, but it's good that you cleaned this up!

Reply:
Oh, I think I still had to use this, it's just in the other file. I also ran into the issue where it wasn't working the other way.